handler/src/main/java/io/netty/handler/ssl/util/SelfSignedCertificate.java

```diff
@@ -416,4 +378,194 @@ private static boolean isBouncyCastleAvailable() {
             return false;
         }
     }
+
+    public static final class Builder {
+        // user fields
+        String fqdn = "localhost";
+        SecureRandom random;
+        int bits = DEFAULT_KEY_LENGTH_BITS;
+        Date notBefore = DEFAULT_NOT_BEFORE;
+        Date notAfter = DEFAULT_NOT_AFTER;
+        String algorithm = "RSA";
```

Comment:
Why not EC?

Reply:
This is the default from the old constructors.

Reply:
Yeah, old API should keep old behavior.

Reply:
Let's change it in 5.x.